### PR TITLE
Added groups_id to createopenslides user command.

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: openslides/users/management/commands/createopenslidesuser.py [hunk 1]
```diff
@@ -26,10 +26,16 @@ def add_arguments(self, parser):
...
+        )
 
     def handle(self, *args, **options):
         user_data = {
             'first_name': options['first_name'],
             'last_name': options['last_name'],
         }
-        User.objects.create_user(options['username'], options['password'], **user_data)
+        user = User.objects.create_user(options['username'], options['password'], **user_data)
+        user.save()
```
Comment (normanjaeckel) [inline openslides/users/management/commands/createopenslidesuser.py]: This save is not necessary.

File: openslides/users/management/commands/createopenslidesuser.py [hunk 2]
```diff
@@ -26,10 +26,16 @@ def add_arguments(self, parser):
...
+        user.groups.add(Group.objects.get(id=options['groups_id']))
```
Comment (normanjaeckel) [inline openslides/users/management/commands/createopenslidesuser.py]: It is probably possbile to use only the id and not to fetch the group from the db.

Comment (normanjaeckel) [inline openslides/users/management/commands/createopenslidesuser.py]: Also you need to validate the input. Only a string that can be transformed to an integer is allowed.

File: openslides/users/management/commands/createopenslidesuser.py [hunk 3]
```diff
@@ -26,10 +26,16 @@ def add_arguments(self, parser):
             'password',
             help='The password of the new user.'
         )
+        parser.add_argument(
+            'groups_id',
+            help='The group ids of the new user.'
```
Comment (normanjaeckel) [inline openslides/users/management/commands/createopenslidesuser.py]: It is only one id.

Comment (emanuelschuetze) [inline openslides/users/management/commands/createopenslidesuser.py]: done

File: openslides/users/management/commands/createopenslidesuser.py (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/users/management/commands/createopenslidesuser.py]: done

File: openslides/users/management/commands/createopenslidesuser.py (hunk 2 above)
Comment (emanuelschuetze) [inline openslides/users/management/commands/createopenslidesuser.py]: done (2x)

**Comment by emanuelschuetze** [comment]
Done

**Pull request merged**